Enrise/accessibility-checker-infrastructure
# Sac-19 Add example page

# How to test

- Run `make init`
- Check http://localhost:1000
- Merge
- Check https://enrise.github.io/accessibility-checker-infrastructure/
